CRM/Event/Form/Participant.php

```diff
@@ -2147,6 +2147,9 @@ protected function sendReceipts($params, array $participants, $lineItem, $additi
       }
     }
     $this->assign('customGroup', $customGroup);
+
+    // @todo - stop setting this, call the function, as appropriate. This is in a weird place.
```

Comment:
Is this `todo` still relevant @eileenmcnaughton ?

Reply:
@colemanw erm no

Reply:
fixed